Exercises/arrays-methods/arrays-methods.js

```diff
@@ -0,0 +1,24 @@
+// Task #1
+const array = [1, 2, 3, 4];
+const isElementPresent = (ar, element) => ar.includes(element);
+console.log(isElementPresent(array, 3));
+
+// Task #2
+const array = [2, 4];
+const isArrayEven = (ar) => !ar.some(item => item % 2 === 1).length;
+console.log(isArrayEven(array));
```

Comment:
`some` and `every` methods return you boolean. `length` of boolean is always undefined.